test/unittest/unittest_nntrainer_activations.cpp

```diff
@@ -517,6 +517,59 @@ TEST(nntrainer_activation, eluPrime_01_p) {
   }
 }
 
+TEST(nntrainer_activation, mish_01_p) {
+  int batch = 3;
+  int channel = 1;
+  int height = 1;
+  int width = 10;
+
+  float answer[30] = {
+    -0.18891649, -0.15113318, -0.10714479, -0.05678857, 0.00000000,
+    0.06317942,  0.13259898,  0.20800139,  0.28903052,  0.37524524,
+    -0.28396326, -0.24693604, -0.18891649, -0.10714479, 0.00000000,
+    0.13259898,  0.28903052,  0.46613652,  0.65969974,  0.86509836,
+    -0.30883580, -0.29565641, -0.24693604, -0.15113318, 0.00000000,
+    0.20800139,  0.46613652,  0.76120591,  1.07794595,  1.40337825};
```

Comment:
quick question. how are you getting this answer data? PyTorch, TensorFlow, or hand calculation?

Reply:
With PyTorch!
I got the answer data by using `torch.nn.Mish()`.